src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int AMOUNT_OF_STORAGE = 10;
+    private static final int NOT_FIND = -1;
+    private Object[] objKey;
+    private Object[] objValue;
```

Comment:
Array names should be plural
```suggestion
    private Object[] objKeys;
    private Object[] objValues;
```
or
```suggestion
    private Object[] keys;
    private Object[] values;
```